README.md

```diff
@@ -1,2 +1,57 @@
+[![Build Status](https://travis-ci.org/dockstore/data-object-service-plugin.svg?branch=master)](https://travis-ci.org/dockstore/data-object-service-plugin)
+[![Coverage Status](https://coveralls.io/repos/github/dockstore/data-object-service-plugin/badge.svg?branch=master)](https://coveralls.io/github/dockstore/data-object-service-plugin?branch=master)
+
 # data-object-service-plugin
-Dockstore Data Object Service file provisioning plugin
+Dockstore Data Object Service file preprovisioning plugin
+
+## Usage
+
+The Data Object Service plugin is capable of calling other Dockstore plugins (i.e. syn, s3, icgc, etc.) to download, upload, and set metadata on uploaded objects.
```

Comment:
I think this description is combining the functionality with the implementation, and they should be broken out. It currently sounds a little like the purpose of the plugin is to call other Dockstore plugins, whereas its purpose is to fetch data via DOS URIs :)

I think you should say something like, although I'm not wild about my wording:

The Data Object Service plugin fetches a data object from a DOS URI, in order to download one the URLs in the data object.

For example, if the data object for a given DOS URI references s3 and gcs URIs, the URL for the s3 data object will be passed on to be downloaded by the s3 plugin, as currently there is not a gcs file plugin for Dockstore.

The plugin only supports downloads. Support for uploads will be added later.

Reply:
How does this sound?

"The Data Object Service plugin fetches data objects from a provided DOS URI in order to download one of the returned URLs.

For example, if the data object for a given DOS URI references s3 and gcs URIs, the URL for the s3 data object is passed into the s3 plugin for downloading, because a gcs file plugin for Dockstore does not currently exist.

The plugin only supports downloads. Support for uploads will be added later."

Reply:
Sounds good!